partner_address_version/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright 2018 Akretion - Benoît Guillot
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': 'Partner Address Version',
+    'version': '10.0.1.0.0',
+    'author': "Akretion, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com',
```

Comment:
@bguillot Replace URL as per [this](https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L11)